services/user.py

```diff
@@ -0,0 +1,36 @@
+from db.models import User
+
+
+def create_user(username: str, password: str, email: str = None,
+                first_name: str = None, last_name: str = None) -> None:
+    user = User(username=username)
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    if email:
+        user.email = email
+    user.set_password(password)
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(pk=user_id)
```

Comment:
better to use `get_user_model()` in this case 

please read https://github.com/mate-academy/py-tickets-and-orders/blob/master/checklist.md